modules/selenium/build.gradle

```diff
@@ -3,8 +3,10 @@ description = "Testcontainers :: Selenium"
 dependencies {
     compile project(':testcontainers')
 
-    provided 'org.seleniumhq.selenium:selenium-remote-driver:3.141.59'
-    provided 'org.seleniumhq.selenium:selenium-chrome-driver:3.141.59'
+    compileOnly 'org.seleniumhq.selenium:selenium-remote-driver:3.141.59'
+    compileOnly 'org.seleniumhq.selenium:selenium-chrome-driver:3.141.59'
+    testCompile 'org.seleniumhq.selenium:selenium-remote-driver:3.141.59'
+    testCompile 'org.seleniumhq.selenium:selenium-chrome-driver:3.141.59'
```

Comment:
This is hopefully the only change that is in any way contentious - just dropping use of the nebula 'provided' plugin and using `compileOnly`/`testCompile` for the Selenium dependencies. I think this provides parity.

Reply:
`provided` should end up in `POM`, while `compileOnly` won't. This is why you had to change `docs/examples/junit4/generic/build.gradle`

I will check what options do we have to recreate the `provided` behaviour, similar to how we do it in Spring projects